libtsuba/src/tsuba.cpp

```diff
@@ -91,8 +85,17 @@ tsuba::Open(const std::string& rdg_name, uint32_t flags) {
     return manifest_res.error();
   }
 
-  return RDGHandle{
-      .impl_ = new RDGHandleImpl(flags, std::move(manifest_res.value()))};
+  return std::move(manifest_res.value());
+}
+
+katana::Result<tsuba::RDGHandle>
+tsuba::Open(RDGManifest rdg_manifest, uint32_t flags) {
```

Comment:
There's probably a discussion to be had about whether `Open` and `RDGHandle` make sense anymore. But not something that we necessarily have to address here.